lib/windows/app/index.js

```diff
@@ -61,6 +61,18 @@ Module._load = function load(path) { // eslint-disable-line no-underscore-dangle
             return ReactDOM;
         case 'react-redux':
             return ReactRedux;
+        case 'nrfconnect/logger':
+            return api.logger;
+        case 'nrfconnect/pc-ble-driver-js':
+            return api.bleDriver;
```

Comment:
My opinion: Make the change. It sounds more user friendly, and user friendly should be preferred. At least if the dev cost is low.